### PR TITLE
Tenzin/unit testing

### PR DESCRIPTION
**Pull request by tenzin1308**

### CODE REVIEW

**Comment by yywu123** [comment]
Good job on fixing the bug that we might facing.

**Comment by bhusalanil121** [comment]
Good start on unit testing.

**Comment by ValeryiaPilchuk** [comment]
Great job on cleaning up code and adding some tests. Have you thought about adding more logging statements?

File: package.json [hunk 1]
```diff
@@ -11,7 +11,8 @@
   "scripts": {
     "test": "echo \"Error: no test specified\"&& exit 1",
     "start": "node server.js",
-    "heroku-postbuild": "NPM_CONFIG_PRODUCTION=false npm install --prefix client && npm run build --prefix client"
```
[the remark this answers is not in the document]
Comment (alicechen1998) [reply package.json:14]: unit test on react?

**Pull request merged**